kube_apiserver_metrics/tests/test_sli_metrics.py

```diff
@@ -0,0 +1,183 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+
+import mock
+import pytest
+import requests_mock
+
+from datadog_checks.kube_apiserver_metrics import KubeAPIServerMetricsCheck
+
+from .common import HERE
+
+# Constants
+CHECK_NAME = "kube_apiserver"
+
+
+@pytest.fixture()
+def mock_metrics():
+    f_name = os.path.join(HERE, "fixtures", "metrics_slis_1.27.3.txt")
+    with open(f_name, "r") as f:
+        text_data = f.read()
+    with mock.patch(
+        "requests.get",
+        return_value=mock.MagicMock(
+            status_code=200,
+            iter_lines=lambda **kwargs: text_data.split("\n"),
+            headers={"Content-Type": "text/plain"},
+        ),
+    ):
+        yield
+
+
+def test_check_metrics_slis(aggregator, mock_metrics, mock_request, instance):
+    mock_request.get("http://localhost:6443/metrics/slis", status_code=200)
```

Comment:
Do you need `status_code=200` here? I'm not sure I follow if that's already provided by `mock_metrics` fixture.

Reply:
you're right it is, removed!